packages-exp/auth-exp/src/core/strategies/custom_token.test.ts

```diff
@@ -67,31 +75,99 @@ describe('core/strategies/signInWithCustomToken', () => {
   });
   afterEach(mockFetch.tearDown);
```

Comment:
Now that you're using `sinon.spy()`, we'll need to add a `sinon.restore()` call to the `afterEach` hook

Reply:
Ooh, nice, done!